tools/SetupFlow/DevHome.SetupFlow.ElevatedComponent/ElevatedComponentOperation.cs

```diff
@@ -14,6 +14,13 @@ namespace DevHome.SetupFlow.ElevatedComponent;
 /// </summary>
 public sealed class ElevatedComponentOperation : IElevatedComponentOperation
 {
+    /// <summary>
+    /// Tasks arguments are passed to the elevated process as input at launch-time.
+    /// </summary>
+    /// <remarks>
+    /// This object is used to ensures that the operations performed at runtime by the
+    /// caller process were pre-approved.
+    /// </remarks>
     private readonly TasksArguments _tasksArguments;
```

Comment:
Nit: I'm not really sure we should be calling these "arguments" after we read them since we want to make it clear that they have been parsed. Not too important, though.